packages/vscode/package.json

```diff
@@ -84,7 +90,7 @@
             "verbose"
           ],
           "default": "off",
-          "description": "Traces the communication between VS Code and the language server."
+          "description": "Setting for logging between the VS Code extension and the language server."
```

Comment:
What does this mean?

Reply:
I don't know if it's even implemented after a quick check 🤔

Reply:
Actually it's a native thing https://code.visualstudio.com/api/language-extensions/language-server-extension-guide#logging-support-for-language-server

Reply:
Good for you?

Reply:
Let's add that as a comment here, so next time we don't have similar questions again - then this should be good.